addons/website/static/src/snippets/s_searchbar/000.js

```diff
@@ -113,7 +113,13 @@ publicWidget.registry.searchBar = publicWidget.Widget.extend({
         res.results.forEach(record => {
             for (const fieldName of fieldNames) {
                 if (record[fieldName]) {
-                    record[fieldName] = Markup(record[fieldName]);
+                    if (typeof record[fieldName] === "object") {
+                        for (const fieldKey of Object.keys(record[fieldName])) {
+                            record[fieldName][fieldKey] = Markup(record[fieldName][fieldKey]);
```

Comment:
Just to make sure I follow correctly:
- this is OK to identify as markup because we know all dict entries were sent through `escape(get_mapping_value(...))` on the python side
- this includes the special `extra_link_title` entry because it got correctly rendered/escaped as everything else and therefore no translation of 'Categories: ' can be used to inject HTML

Reply:
Yes, you're right ! :)